sys/kern/src/arch/rv64/mod.rs

```diff
@@ -10,6 +10,9 @@ pub use macros::*;
 pub mod trap;
 pub use trap::*;
 
+pub mod task;
```

Comment:
unrelated change?

Reply:
Negative. Lines from a@22, a@23 had these previously, they got reordered to move them out of the middle of a cfg_if block

Reply:
Ack. The cfg_if comes in a different change, so, it was a little confusing. But, I see what you are saying when I look at the final combined change.